beacon-chain/forkchoice/doubly-linked-tree/proposer_boost_test.go

```diff
@@ -156,7 +156,7 @@ func TestForkChoice_BoostProposerRoot_PreventsExAnteAttack(t *testing.T) {
 		// Ancestors have the added weights of their children. Genesis is a special exception at 0 weight,
 		require.Equal(t, f.store.treeRootNode.weight, uint64(0))
 
-		// Proposer boost score with this tests parameters is 8
+		// Proposer boost score with these tests parameters is 8
```

Comment:
```suggestion
		// Proposer boost score with these test parameters is 8
```

Reply:
Plural refers to the parameters here, not the tests

Reply:
Makes sense. Thanks for the clarification